pkg/util/alertmanager/alertmanager.go

```diff
@@ -0,0 +1,84 @@
+package alertmanager
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net"
+	"net/http"
+
+	"github.com/prometheus/common/model"
+	"github.com/sirupsen/logrus"
+	"k8s.io/client-go/rest"
+
+	"github.com/Azure/ARO-RP/pkg/util/portforward"
+)
+
+type AlertManager interface {
+	FetchPrometheusAlerts(ctx context.Context) ([]model.Alert, error)
+}
+
+type alertManager struct {
+	log        *logrus.Entry
+	restConfig *rest.Config
+}
+
+func NewAlertManager(c *rest.Config, log *logrus.Entry) AlertManager {
+	return &alertManager{log, c}
+}
+
+func (a *alertManager) FetchPrometheusAlerts(ctx context.Context) ([]model.Alert, error) {
```

Comment:
Since the purpose of this PR is focusing on the Admin Portal feature, we can address the use of `util` code for monitor in another PR, we have created a separate work item for the same here. 
[Switch Monitor to use the new alert-manager interface](https://issues.redhat.com/browse/ARO-1850 )